src/IECoreMaya/SceneShapeInterface.cpp

```diff
@@ -824,6 +825,34 @@ MStatus SceneShapeInterface::compute( const MPlug &plug, MDataBlock &dataBlock )
 					{
 						MFnNurbsCurveData fnData;
 						data = fnData.create();
+
+						// If "index" convert paramter is specified, it determines which one of the curves to output in the Curves primitive.
+						std::map<std::string, std::string> convertParams;
+						bool isParamsRead = readConvertParams( convertParams, index );
```

Comment:
I think it'll be better keep using a string unless we want to hide the information from the artists but there's a chance they want IECore::Objects depending on the speed. In that case we will come back and think about it.